decidim-core/app/helpers/decidim/newsletters_helper.rb

```diff
@@ -44,5 +32,50 @@ def custom_url_for_mail_root(organization, newsletter_id = nil)
     def utm_codes(host, newsletter_id)
```

Comment:
Just finishing the documentation I saw that this method is public, but it could be private, as its only being used by the `custom_url_for_mail_root` method. Can you move it to the private section?